01week/resume/css/style.css

```diff
@@ -0,0 +1,107 @@
+body {
+  background-image: url("http://freebie.photography/background/slides/wires.jpg");
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+a {
+  color: orange;
+}
+
+#container {
+  background-color: white;
+  width: 900px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 10px;
+}
+
+h1 {
+  margin: 0;
+}
+
+header {
+  background-color: rgb(255, 182, 45);
+  color: white;
+  text-align: center;
+  margin: 0;
+  padding: 24px;
+  border-radius: 10px 10px 0 0;
+}
+
+nav {
+  background-color: gray;
+  color: white;
+  text-align: center;
+  margin: 0;
+  padding: 24px;
+}
+
+nav .selected {
+  font-weight: bold;
+}
+
+nav ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+#trinetlogo {
+  position: relative;
+  left: 100px;
+  top: 50px;
+}
+
+#embarcadero {
+  position: relative;
+  left: 65px;
+  top: 70px;
+}
+
+#informatica {
+  position: relative;
+  left: 60px;
+  top: 80px;
+}
+
+#apple {
+  position: relative;
+  left: 110px;
+  top: 80px;
+}
+
+#acalogo {
```

Comment:
you see how these two styles are exactly the same?

you should setup a class instead of an id when the style can be applied to more then one thing

Reply:
Yes - I did some research on using classes and applied them to my bio page
Thanks!